config.yaml

```diff
@@ -0,0 +1,17 @@
+sources:
```

Comment:
Should also `git rm` this one. Probably was committed by accident.

Reply:
Oops, fixed